addon/ng2/models/webpack-build-production.ts

```diff
@@ -15,7 +15,6 @@ export const getWebpackProdConfigPartial = function(projectRoot: string, appConf
     },
     plugins: [
       new WebpackMd5Hash(),
-      new webpack.optimize.DedupePlugin(),
       new webpack.optimize.UglifyJsPlugin(<any>{
         mangle: { screw_ie8 : true, keep_fnames: true },
```

Comment:
Thanks @TheLarkInn I'll strip that out and re-push, but I don't expect e2e to work based upon that change